src/lib.rs

```diff
@@ -494,32 +491,31 @@ pub mod subscrypt {
         /// Examples in `withdraw_works` and `withdraw_works2` in `tests/test.rs`
         #[ink(message)]
         pub fn withdraw(&mut self) -> u128 {
+            // TODO match & unwrap here
```

Comment:
Sorry  @mmagician , I think the current implementation is simple and clean enough.
I'm not sure that I understood your idea to enhance the current code.

Reply:
Makes sense